Gopkg.lock

```diff
@@ -306,6 +307,12 @@
   ]
   revision = "37ab263305aaeb501a60eb16863e808d426e37f2"
 
+[[projects]]
+  name = "github.com/hashicorp/logutils"
```

Comment:
just wondering about why hashicorp logutils? is this providing anything extra?

Reply:
@dipak-pawar This is a dependency of pact-go.

Reply:
but I can see it's been used in https://github.com/fabric8-services/fabric8-wit/pull/2393/files#diff-519a11ea41b1a50fe2e34ad14348a61eR44

Reply:
You're right, that is/was a hack to suppress pact publisher's DEBUG output. I removed it and replaced it with a more elegant way.